spec/stack_length_test.js

```diff
@@ -10,3 +10,14 @@ describe('Stack', () => {
 it('exists', () => {
```

Comment:
All tests for the same function would be better in the same file.
Feel free to reach out to me for clarification or questions about chai